specification/src/main/asciidoc/platform/Profiles.adoc

```diff
@@ -136,7 +138,7 @@ requirements.
 [[a3240]]
 === Requirements for All Jakarta EE Profiles
 
-The Java Platform, Standard Edition 8 is the
+The Java Platform, Standard Edition 11 is the
 required API foundation for any Jakarta EE profile.
```

Comment:
I would replace the word "foundation" with "compile level". Foundation feels a bit too open for interpretation, where one could read it as compile and runtime level.